src/game/BattleGround/BattleGroundMgr.cpp

```diff
@@ -1251,22 +1251,16 @@ void BattleGroundMgr::BuildPvpLogDataPacket(WorldPacket* data, BattleGround* bg)
     if (type)                                               // arena
     {
         // it seems this must be according to BG_WINNER_A/H and _NOT_ BG_TEAM_A/H
-        for (int8 i = 1; i >= 0; --i)
-        {
-            *data << uint32(3000 - bg->m_ArenaTeamRatingChanges[i]);                    // rating change: showed value - 3000
-            *data << uint32(3999);                          // huge thanks for TOM_RUS for this!
-            DEBUG_LOG("rating change: %d", bg->m_ArenaTeamRatingChanges[i]);
```

Comment:
I don't see why we'd remove this logging. In fact, isn't this something we would like the server to log itself as this is somewhat valuable info to whom might run their own server - I don't see why this would be debug info at all, though?